torchaudio/models/decoder/_ctc_decoder.py

```diff
@@ -31,7 +33,16 @@
         Dictionary as _Dictionary,
         load_words as _load_words,
     )
-except Exception:
+
+    try:
+        from flashlight.lib.text.decoder import KenLM as _KenLM
```

Comment:
Any reason you wouldn't want to change this to `flashlight.lib.text.decoder.kenlm` once I merge things in if you're already changing this stuff as is? Given that these are the first PyPI releases we've done, I don't care about breaking things on our side this early.

Reply:
From the strict backward compatibility perspective, this "if flashlight is available import from there" logic existed for users who installed flashlight from the source before PyPI release, and this initialization logic never worked for PyPI release, as it lacks KenLM, it is preferrable to keep the same naming sematics. I am fine with changing it to the new one, but that means, we ask users who installed FL Text from source to upgrade.

Reply:
BTW I just realized that this logic is used in fbcode, so to change the KenLM naming convention to `flashlight.lib.text.decoder.kenlm`, then I think the corresponding part of torchaudio has to be changed at the same time in fbcode.

Reply:
Right -- as an alternative, could we potentially try to import from both places and show a deprecation warning if the import with this method succeeds? That way, things can still work in fbcode once everything is changed.